mu-plugins/blocks/global-header-footer/header.php

```diff
@@ -32,6 +38,14 @@
 	</figure>
 	<!-- /wp:image -->
 
+	<?php if ( ! empty( $locale_title ) ) : ?>
+	<!-- wp:paragraph {"className":"global-header__wporg-local-title"} -->
```

Comment:
```suggestion
	<!-- wp:paragraph {"className":"global-header__wporg-locale-title"} -->
```

`local` is used a couple other places. If it's intentional, though, that WFM

Reply:
Seems like it was a typo, I've changed it to locale.